### DISCUSSION TITLE
A Logo 15 Word Design

### DISCUSSION THREAD

**Discussion by dmalec** [jrincayc/ucblogo-code#77] [Show and tell]
After looking at the entries in [The Logo 15-word challenge](http://www.mathcats.com/gallery/15wordcontest.html), I revisited one of my plot designs and came up with the following version using 15 words:
```logo
repeat 100 [ make "s repcount repeat 3 [ forward :s left 118.2 ] right 33.3 forward repcount ]
```

Which renders as:
![TriangularPinwheel_ScreenTurtle](https://user-images.githubusercontent.com/330202/102142913-25987480-3e31-11eb-8a55-8e50126b6f6b.png)

I then used `epspict` to export the drawing, imported it into Inkscape, and, after a bit of scaling and positioning, plotted it in bronze ink on black paper using an AxiDraw:
![TriangularPinwheel_AxiDrawPlot](https://user-images.githubusercontent.com/330202/102142905-229d8400-3e31-11eb-88bf-c75ca1533acb.png)

`118.2` and `33.3` were the values I personally liked best; but, there's a number of variations in those two numbers which produce interesting results.

**Comment by ghost** [comment]
Cool. I should decorate my office with my own Logo art. In fact, I think I will!

**Reply by dmalec** [reply]
Excellent plan! I have a few of my favorite Logo designs up in mine :)

If you're willing to post/share some of your Logo art, I'd be interested in seeing them :)
